README.md

```diff
@@ -0,0 +1,28 @@
+# ginger-algebra-ocl
```

Comment:
I think it will be nice and important too to add instructions about the building process and dependencies required

Reply:
I've added more details to the README and also I have put the additional unit-tests directly to the ginger-algebra-ocl, which can detect if the GPU accessible on the machine and configured properly for this library